patientsearch/src/js/components/PatientListTable.js

```diff
@@ -1093,6 +1095,11 @@ export default function PatientListTable() {
           return false;
         }
         if (appSettings) {
+          addMamotoTracking(
+            appSettings["MATOMO_SITE_ID"],
+            // use preferred_username from token as user id
```

Comment:
Just a small change to remove some ambiguity between OIDC/SoF and Access/ID token (These attributes are provided by flask-oidc from decoded JWTs)
```suggestion
            // use preferred_username from OIDC ID token as user id
```